<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Window</class>
 <widget class="QWidget" name="Main_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Маркировка кодами ЧЕСТНЫЙ ЗНАК</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>221</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnOrders">
      <property name="text">
       <string>Обработка заказов</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPrintDatamatrix">
      <property name="text">
       <string>Печать этикеток с кодами DataMatrix</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPrintSSCC">
      <property name="text">
       <string>Печать штрихкодов для коробов</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAgregation">
      <property name="text">
       <string>Агрегация в короба</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnClose">
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
